src/linagora.esn.calendar/app/calendar/calendar.controller.js

```diff
@@ -14,6 +14,7 @@
         self.uiConfig = calendarUiConfiguration;
         self.uiConfig.calendar.businessHours = self.businessHours;
         self.uiConfig.calendar.scrollTime = self.businessHours[0].start;
+        self.isMobile = /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent);
```

Comment:
you can use detectUtils for this

```suggestion
        self.isMobile = detectUtils.isMobile();
```